composer.json

```diff
@@ -9,15 +9,14 @@
     ],
     "require": {
         "php": ">=7.0.0",
-        "silverstripe/recipe-plugin": "^1.3",
```

Comment:
Does removing this still result in 1.3 being installed? Recipe-cms includes ^1.2